lib/add-cors-headers.js

```diff
@@ -31,7 +31,7 @@ module.exports = function addCorsHeaders (request, reply) {
     }
   }
 
-  response.headers['access-control-allow-origin'] = request.headers.origin
+  response.headers['access-control-allow-origin'] = request.headers.origin || "*"
```

Comment:
problem is that "*" is not allowed when `access-control-allow-credentials` is set to true